great_ape_safe/ape_api/citadel.py

```diff
@@ -78,12 +78,24 @@ def get_citadel_distribution_split(self):
             "locking": "{:.2%}".format(self.citadel_minter.lockingBps() / self.MAX_BPS),
         }
 
-    def set_citadel_distribution_split(self, funding_bps, staking_bps, locking_bps):
-        # enforce here so there is not revert at SC level
-        assert funding_bps + staking_bps + locking_bps == self.MAX_BPS
-        self.citadel_minter.setCitadelDistributionSplit(
-            funding_bps, staking_bps, locking_bps
-        )
+    def set_citadel_distribution_split(
+        self, funding_bps, staking_bps, locking_bps, dao_bps=None
+    ):
+        if not dao_bps:
+            # enforce here so there is not revert at SC level
+            assert funding_bps + staking_bps + locking_bps == self.MAX_BPS
+            self.citadel_minter.setCitadelDistributionSplit(
+                funding_bps, staking_bps, locking_bps
+            )
```

Comment:
Maybe have two versions of this script; one for the current beta version of the contract, and one for the final version?

Once live, I don't think it will make sense to keep this part of the script around, plus you would constantly have to pass `dao_bps=True` to the script...

Reply:
at the end i just adapted the script to latest minter script, so only one version of the script